_schedule/talks/2019-09-23-11-30-t0-awesome-automated-apis-with-automagic.md

```diff
@@ -35,4 +35,4 @@ Databases stay relevant by continuing to reinvent themselves to serve new techno
 
 But what do these relations look like in practice? In this talk, I will show how our team at The Wharton School has evolved from providing financial data exclusively in SAS data formats to a robust backend powered by PostgreSQL, which allows financial research to happen in many ecosystems: still available in SAS, but also R, Python, Perl, Matlab, Julia, and more. A case study will be shown that uses Django REST Framework to build an "API through introspection." This case study will show how WRDS built a Django web site and RESTful web service by introspecting financial data stored in a PostgreSQL database cluster.
 
-The models for the ORM, serializers for the RESTful API, views for presenting the data to a user, filters for refining queries, URL routing, web browsable interface, user token authorization, and permissions, are all handled by introspecting various features of the PostgreSQL database information schema, all with Python. Many of these components have been open-sourced, such as the ability to export an endpoint as a spreadsheet file (XLSX), and a generic automated API builder, called Automagic REST.
\ No newline at end of file
+The models for the ORM, serializers for the RESTful API, views for presenting the data to a user, filters for refining queries, URL routing, web browsable interface, user token authorization, and permissions, are all handled by introspecting various features of the PostgreSQL database information schema, all with Python. Many of these components have been open-sourced, such as the ability to export an endpoint as a spreadsheet file (XLSX), and a generic automated API builder, called Automagic REST.
```

Comment:
what changed here?

Reply:
Basically https://editorconfig.org/

We have a config that controls how files get saved. Hopefully, someone adds github support and then we don't have to worry about editors doing weird things.